lib/MusicBrainz/Server/Controller/Admin.pm

```diff
@@ -161,15 +161,22 @@ sub email_search : Path('/admin/email-search') Args(0) RequireAuth(account_admin
     my ($self, $c) = @_;
 
     my $form = $c->form(form => 'Admin::EmailSearch');
-    my @results;
-    my $searched = 0;
+    my $results;
+    my $stored_email = $c->session->{searched_email};
 
     if ($c->form_posted_and_valid($form, $c->req->body_params)) {
         try {
```

Comment:
Maybe we should `delete $c->session->{searched_email};` here so that if an error occurs it doesn't revert to the previously saved search when you reload the form?

Reply:
I guess that would make sense, will do.